R/rebuild.R

```diff
@@ -34,7 +34,9 @@ reset_search_path <- function() {
 }
 
 attach <- function(locked_package) {
-  if (!identical(locked_package$load, FALSE) && !locked_package$is_dependency_package) {
+  if (is.autoinstall_package(locked_package)) {
```

Comment:
Ok load FALSE will overwrite autoinstall TRUE now.

I can't get the last one to work because there's no autoinstall-specific class.